CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # Change Log
+
+## [7.54.0](https://github.com/plivo/plivo-go/tree/v7.54.0) (2024-10-28)
+**Feature - GetRecordingTranscription feature to get transcription**
+- Support for the `type` filter parameter, supported filters are transcription, raw and diarized
```

Comment:
How about the other 2  requirements in the JIRA ?

Participant recording transcription params
MPC recording transcription params